superduperdb/container/model.py

```diff
@@ -520,20 +522,16 @@ def _validate(
             validation_set = t.cast(Dataset, db.load('dataset', validation_set))
 
         mdicts = [MongoStyleDict(r.unpack()) for r in validation_set.data]
-        # TOOD: self.train_X, self.train_y are sequences of strings: this can't work
-        prediction_X = self._predict(
-            [d[self.train_X] for d in mdicts]  # type: ignore[index]
-        )
-        prediction_y = self._predict(
+        prediction = self._predict(
             [d[self.train_X] for d in mdicts]  # type: ignore[index]
         )
-        assert isinstance(prediction_X, list)
-        assert isinstance(prediction_y, list)
-        assert all(isinstance(i, int) for i in prediction_X + prediction_y)
+        target = [d[self.train_y] for d in mdicts]  # type: ignore[index]
+        assert isinstance(prediction, list)
+        assert isinstance(target, list)
```

Comment:
it always will be list? what's the point of is instance

Reply:
I think this is to suppress `mypy` warnings (this was from the previous commit).

Reply:
ok..

Reply:
ignore[index] could be remove by asserting self.train_y as string?